app/webpacker/components/Tickets/TicketWorkbenches/CompetitionResultActionerView/TimelineView.jsx

```diff
@@ -19,6 +19,17 @@ const TIMELINE_STATUSES = [
     description: `WRT will be shown the list of warnings and the message from
     Delegate. WRT needs to review them and mark it as done.`,
   },
+  {
+    status: ticketsCompetitionResultStatuses.merged_temporary_results,
+    description: `WRT will have a rough look at the results to make sure that
+    there is no major flaw that requires aborting the posting process. For
+    example, check if there are big suspicious chunks of DNFs/DNSs and verify
+    that the shape of the results is sound (e.g. results should generally grow
+    “wider” from top to bottom with very few exceptions like DNFs or cutoffs).
+    Once done with the rough look, proceed to click the “Merge Temporary
+    Results” button which will copy data from InboxResults to Results, then
+    clear the data in InboxResults.`,
```

Comment:
Did you have such a description before? Or did you just come up with it now for the PR?

If it's the latter: Where will the description be shown, and who will usually read it?

Reply:
I copied this description from WRT procedures doc. I have a timeline view on the top of ticket workbench, so WRT members can refer there on the step. Eventually this can even be moved away from procedures doc so that the ticket will be single source of truth, and future new members will find it more easier.